crl/crl.go

```diff
@@ -0,0 +1,42 @@
+package crl
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/letsencrypt/boulder/issuance"
+)
+
+// number must be constructed by calling `NewNumber`. It represents the
+// 'crlNumber' field of a CRL.
+type number *big.Int
+
+// Number derives the 'CRLNumber' field for a CRL from the value of the
+// 'thisUpdate' field provided in Unix nanoseconds.
+func Number(thisUpdate int64) number {
+	// Per RFC 5280 Section 5.2.3, 'CRLNumber' is a monotonically increasing
+	// sequence number for a given CRL scope and CRL that MUST be at most 20
+	// octets. A 64-bit (8-byte) integer will never exceed that requirement but
+	// let's guarantee this.
```

Comment:
"let's guarantee this" seems like a mis-paraphrase of "lets us guarantee this" in the old ca/crl.go. The previous comment was talking about how we pack ThisUpdate (a timestamp) into a 64-bit integer, which guarantees an increasing sequence number while not exceeding the 20 octets available to us.

In that spirit, I think the right interface for this function is a `time.Time` instead of an int64, since that better expresses the intent that we derive CRL numbers from timestamps.